cvat-ui/src/actions/requests-async-actions.ts

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2024 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { ThunkAction } from 'utils/redux';
+import {
+    CombinedState,
+    RequestsQuery, StorageLocation,
+} from 'reducers';
+import {
+    getCore, RQStatus, Request, Project, Task, Job,
+} from 'cvat-core-wrapper';
+import { exportBackupAsync, exportDatasetAsync } from './export-actions';
+import { RequestInstanceType, listen, requestsActions } from './requests-actions';
+import { importDatasetAsync } from './import-actions';
+
+const core = getCore();
+
+export interface RequestParams {
+    id: string;
+    type: string;
+    instance?: Project | Task | Job;
+    location?: StorageLocation;
+}
+
+export function getRequestsAsync(query: RequestsQuery, notify = true): ThunkAction {
+    return async (dispatch, getState): Promise<void> => {
+        dispatch(requestsActions.getRequests(query));
+
+        const state: CombinedState = getState();
+
+        try {
+            const requests = await core.requests.list();
+            dispatch(requestsActions.getRequestsSuccess(requests));
+
+            if (notify) {
+                const shownNotifications = JSON.parse(localStorage.getItem('requestsNotifications') || '[]');
```

Comment:
0. Parsing item from local storage should be try catched, otherwise now it will get into global try catch and show "get requests failed message". That is not true. 
1. Local storage is limited by 5 MB, we will quickly make it full with big number of request (as I see you only append something inside. But never remove. So, gradually it will be full)
2. Finally, I believe we may totally omit this logic. Thus, a user will only see notification if the process finished when the page is opened. I believe it is enough.

Reply:
Removed this logic